hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/pbj/PbjConverter.java

```diff
@@ -1504,7 +1504,7 @@ public static com.hederahashgraph.api.proto.java.FixedFee fromPbj(@Nullable Fixe
         var builder = com.hederahashgraph.api.proto.java.FixedFee.newBuilder();
         if (fixedFee != null) {
             builder.setAmount(fixedFee.amount());
-            builder.setDenominatingTokenId(fromPbj(fixedFee.denominatingTokenId()));
+            builder.setDenominatingTokenId(fromPbj(fixedFee.denominatingTokenIdOrElse(TokenID.DEFAULT)));
```

Comment:
Instead of this we should add 
```
if(fixedFee.hasDenominatingTokenID()){
builder.setDenominatingTokenId(fromPbj(fixedFee.denominatingTokenId()));
}

Reply:
Done